test/integration/suites-k8s/k8s/00-setup

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I'm not sure if there is a good reason to have this in a completely separated suite. I would probably try to have this as part of the existing k8s integration test.